src/test/features/response/form/models/timelineBrakedown.ts

```diff
@@ -0,0 +1,133 @@
+import { expect } from 'chai'
+
+import { TimelineRow } from 'response/form/models/timelineRow'
+import { INIT_ROW_COUNT, TimelineBreakdown } from 'response/form/models/timelineBreakdown'
+
+describe('TimelineBreakdown', () => {
+
+  describe('initialRows', () => {
+
+    it('should return array of empty instances of TimelineRow', () => {
+
+      const actual: TimelineRow[] = TimelineBreakdown.initialRows()
+
+      expect(actual.length).to.equal(4)
+      expectAllRowsToBeEmpty(actual)
+    })
+  })
+
+  describe('fromObject', () => {
+
+    [false, null, undefined, 0, '', NaN].forEach(value => {
```

Comment:
As far as I remember we wanted to avoid coding against `null` in TS code, so I guess it can be removed from this list?

Reply:
ok